File: src/main/java/org/taktik/icure/services/external/rest/v1/dto/embed/RenewalDto.java
package org.taktik.icure.services.external.rest.v1.dto.embed;

import java.io.Serializable;

public class RenewalDto implements Serializable {

    Integer decimal;
    DurationDto duration;

    public Integer getDecimal() { return decimal; }

    public void setDecimal(Integer decimal) { this.decimal = decimal; }

    public DurationDto getDuration() { return duration; }

    public void setDuration(DurationDto duration) { this.duration = duration; }
}

File: src/main/kotlin/org/taktik/icure/services/external/rest/v1/dto/embed/RenewalDto.kt
package org.taktik.icure.services.external.rest.v1.dto.embed

import com.github.pozo.KotlinBuilder
import java.io.Serializable

@KotlinBuilder
data class RenewalDto(
        val decimal: Int? = null,
        val duration: DurationDto? = null
) : Serializable
